content/examples/enterprise-quickstart/docker-compose.yaml

```diff
@@ -0,0 +1,69 @@
+version: "3"
+networks:
+  main: {}
+services:
+  pomerium:
+    image: pomerium/pomerium:latest
+    volumes:
+      - ./config.yaml:/pomerium/config.yaml:ro
+    ports:
+      - 443:443
+    networks:
+      main:
+        aliases:
+        - authenticate.localhost.pomerium.io
+  pomerium_console:
+    networks:
+      main:
+    depends_on:
+      database:
+        condition: service_healthy
+      pomerium:
+        condition: service_started
+    image: docker.cloudsmith.io/pomerium/enterprise/pomerium-console:latest
+    expose:
+      - 8701
+      - 9090
+    environment:
+      - AUDIENCE=console.localhost.pomerium.io
+      - ADMINISTRATORS=admin@example.com
```

Comment:
maybe we should externalize all customizable env variables rather then expect the user to spot the "REPLACE ME" https://docs.docker.com/compose/environment-variables/#using-the---env-file--option

Reply:
I think we should deploy this as is. Externalizing customizable variables would require adding a `.env` file with placeholder values. Either way, the user will need to fill in those values themselves, so some configuration is required. Thoughts?